### DISCUSSION TITLE
Feedback for experimental keep-alive support

### DISCUSSION THREAD

**Discussion by rhansen** [nginx-proxy/nginx-proxy#2194] [Ideas]
Pull request #1934 added experimental support for keep-alive between nginx and the upstream (backend) servers.  If you use this feature, or if you would like changes to the feature before you start using it, please provide feedback here so we know whether it is working properly and can be promoted to officially supported.

**Comment by buchdag** [comment]
Since no one chimed in here since march, I think this can be promoted to officially supported, and given [how nginx consider not enabling keepalive connections](https://www.nginx.com/blog/avoiding-top-10-nginx-configuration-mistakes/#no-keepalives), this will probably also be made default with the new [`auto`](https://github.com/nginx-proxy/nginx-proxy/pull/2364) setting.
